pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -141,10 +171,56 @@ func newClaimInfoCache(stateDir, checkpointName string) (*claimInfoCache, error)
 	return cache, nil
 }
 
+// DescribeWithStability implements the metrics.StableCollector interface.
+func (cache *claimInfoCache) DescribeWithStability(ch chan<- *metrics.Desc) {
+	ch <- kubeletmetrics.DRAResourceClaimsInUseDesc
+}
+
+// CollectWithStability implements the metrics.StableCollector interface.
+func (cache *claimInfoCache) CollectWithStability(ch chan<- metrics.Metric) {
+	cache.RLock()
+	defer cache.RUnlock()
+
+	// Only currently known drivers are listed. If a driver had active
+	// claims in the past, no longer does and then gets uninstalled, it no
+	// longer shows up. This avoids the memory leak problem in a normal
+	// GaugeVec which could grow over time unless obsolete drivers are
+	// actively deleted.
+	//
+	// The empty driver name provides the overall count of all active
+	// ResourceClaims regardless of the driver.
```

Comment:
This is up for debate, see https://github.com/kubernetes/kubernetes/pull/131641#discussion_r2077465713